dashboard/test/ui/features/i18n.feature

```diff
@@ -0,0 +1,44 @@
+Feature: See Hour of Code and Frozen tutorial in Spanish, Portuguese, and Arabic
+
+Background:
+  Given I am on "http://studio.code.org/reset_session"
```

Comment:
I think these links still need to be `learn.code.org` to get replaced in https://github.com/code-dot-org/code-dot-org/blob/13049e7c661f183deec9b0565d395dafeb545ca9/dashboard/test/ui/step_definitions/steps.rb#L5